.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Use Unix line endings in all text files.
+* text=auto eol=lf
```

Comment:
This fixes the issue with line endings on windows:
![image](https://github.com/Druue/rps/assets/29753584/2a7ec9c0-d54e-4bd0-b998-4769df0e7f6e)